core-web/libs/portlets/edit-ema/portlet/src/lib/dot-ema-shell/dot-ema-shell.component.ts

```diff
@@ -15,10 +15,14 @@ import { map, skip, take, takeUntil } from 'rxjs/operators';
 import {
     DotESContentService,
     DotFavoritePageService,
+    DotHttpErrorManagerService,
     DotLanguagesService,
     DotPageLayoutService,
     DotPageRenderService,
-    DotPersonalizeService
+    DotPersonalizeService,
+    DotWorkflowActionsFireService,
+    DotWorkflowsActionsService,
+    DotWorkflowEventHandlerService
```

Comment:
This is why modules are for no? Maybe... just a suggestion.

Reply:
I just let the services on the component that needs them.